File: C++/1573A. Countdown.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int t, n;
    cin>>t;
    for(int i=0; i<t; i++){
        cin>>n;
        string s;
        int count=0;
        cin>>s;
        for(int j=0; j<s.size(); j++){
            if(s[s.size()-1]!='0'){
                count = count + (int(s[s.size()-1])- 48);
                s[s.size()-1] = '0';
            }
            if(s[j]!='0'){
                count = count + 1 + (int(s[j])- 48);
            }
        }
        cout<<count<<endl;
    }

    return 0;
}

File: Python/1573A. Countdown.py
def main():
    t = int(input())
    for i in range(t):
        n = int(input())
        count = 0
        s = list(input())
        for j in range(n):
            if s[n-1] != '0':
                count = count + int(s[n-1])
                s[n-1] = '0'
            if s[j] != '0':
                count = count + int(s[j]) + 1
                
        print(count)
main()